roles/openshift_logging/files/elasticsearch-logging.yml

```diff
@@ -28,32 +39,24 @@ logger:
 additivity:
   index.search.slowlog: false
   index.indexing.slowlog: false
+  deprecation: false
 
 appender:
-  console:
```

Comment:
I think we should leave the appender  so we can turn on console logging if we have to.  And if we can make it possible to modify fluentd to ignore in future